frontend/src/components/dashboard/NewsFeedTabContent.tsx

```diff
@@ -104,15 +91,30 @@ export default () => {
   }, [preprocessedActivities]);
 
   const searchResults = useMemo(() => {
-    const filteredActivites = activities?.filter((item) =>
-      filter === "Organizations"
-        ? item.activity_type?.includes("organization")
-        : filter === "Datasets" || filter === "Datasets Approvals"
-        ? item.activity_type?.includes("package")
-        : true
-    );
+    const filteredActivites =
+      categoryFilter === "All"
+        ? activities
+        : activities?.filter((item) =>
+            categoryFilter === "Organizations"
```

Comment:
There is something wrong with these filters. I have three news in my feed, but when I search by any word of the news, they aren't presented.

![image](https://github.com/user-attachments/assets/6653651b-a31e-4a04-8e62-3b6563adc55d)

<img width="862" alt="image" src="https://github.com/user-attachments/assets/770d6059-3da5-48c4-9144-94fa4d758d33">

Reply:
As per the latest changes, the search bar searches for Dataset/Organization name and title and activity actor.
And I have added an actions filter where you can filter by activity action as per the requirement.

Reply:
@muhammad-hassan11 so how should I filter using this filter? I'm not able to do it.

Reply:
<img width="1330" alt="image" src="https://github.com/user-attachments/assets/7c2107e4-97b2-48cf-91e8-06448f16fcf6">